src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/af-page-v2_1/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Template">
+    <jcr:content
+        jcr:description="Generic template for Adaptive Form with core components V2.1"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Adaptive Form V2.1"/>
```

Comment:
title needs to be updated.

Reply:
As per last discussion we are reusing the existing template & template types for headless .
cc :- @rismehta